### PR TITLE
feat(lint): unify LESS files whitespaces and string quotes

### PR DESCRIPTION
**Pull request by mvorisek**
This PR is part of GH-2594 but separated to integrate many whitespace changes (and other non functional changes) separately.

### CODE REVIEW

File: src/definitions/collections/breadcrumb.less [hunk 1]
```diff
@@ -12,8 +12,8 @@
             Theme
 *******************************/
 
-@type : 'collection';
-@element : 'breadcrumb';
+@type: 'collection';
```
Comment (mvorisek) [inline src/definitions/collections/breadcrumb.less:15]: sadly not fixed by `declaration-colon-space-before` rule (but enabled in https://github.com/stylelint/stylelint-config-standard/blob/29.0.0/index.js#L76)

File: src/semantic.less [hunk 2]
```diff
@@ -10,6 +10,8 @@
   Import this file into your LESS project to use Fomantic-UI without build tools
 */
 
+// stylelint-disable no-invalid-position-at-import-rule
```
Comment (mvorisek) [inline src/semantic.less:13]: `& { @import 'definitions/globals/reset'; }` is used to isolate the imported variable scope?

File: src/themes/default/elements/icon.variables [hunk 3]
```diff
@@ -83,1463 +83,1463 @@ Icons are order A-Z in their group, Solid, Outline, Thin (Pro only) and Brand
 /* Deprecated *In/Out Naming Conflict) */
 @icon-deprecated-map: {
     linkedin_in:\f0e1;
-    zoom_in: \f00e;
-    zoom_out: \f010;
-    sign_in: \f2f6;
-    in_cart: \f218;
-    log_out: \f2f5;
-    sign_out: \f2f5;
+    zoom_in: '\f00e';
```
Comment (mvorisek) [inline src/themes/default/elements/icon.variables:86]: fixes https://github.com/fomantic/Fomantic-UI/pull/2594#discussion_r1048969446

the mentioned export/generator lib should be fixed to export the data with single quotes and no space before colon and one space after

File: src/themes/default/modules/dropdown.overrides [hunk 4]
```diff
@@ -35,25 +35,25 @@
 
 .ui.dropdown .item .left.dropdown.icon::before,
 .ui.dropdown .left.menu .item .dropdown.icon::before {
-    content: "\f0d9"/*rtl:"\f0da"*/;
+    content: '\f0d9'/*rtl:'\f0da'*/;
```
Comment (mvorisek) [inline src/themes/default/modules/dropdown.overrides:38]: if https://github.com/fomantic/Fomantic-UI/pull/2594/files/ebc0cf12f5c9a7644db1757fa0815a262a4821bf#r1048963174 is parsed by the build script somehow, please test if single quotes works, both should work

File: src/semantic.less (hunk 2 above)
Comment (lubber-de) [reply src/semantic.less:13]: Yes, according to  9f750c262f832e979337d43c872f8a2c88c7d3b

File: src/definitions/modules/dropdown.less [hunk 5]
```diff
@@ -805,7 +805,7 @@ select.ui.dropdown {
     }
 
     .ui.dropdown select.noselection ~ .remove.icon,
-    .ui.dropdown input[value=''] ~ .remove.icon,
+    .ui.dropdown input[value=""] ~ .remove.icon,
```
Comment (lubber-de) [inline src/definitions/modules/dropdown.less:808]: Please keep double quotes in selectors

File: src/themes/default/elements/icon.variables (hunk 3 above)
Comment (lubber-de) [reply src/themes/default/elements/icon.variables:86]: Done by https://github.com/fomantic/create-fomantic-icons/pull/215

**Review by lubber-de — APPROVED** [review]
LGTM

**Pull request merged**